components/audio/src/vwc-audio.ts

```diff
@@ -152,8 +156,8 @@ export class VWCAudio extends LitElement {
 						type="${this._isPlaying ? 'pause-solid' : 'play-solid'}"
 				></vwc-icon>
 				</button>
-				${this.timestamp ? html`<div class="playhead-position">${formatTime(this._playheadPosition)} / ${formatTime(this._duration)}</div>` : nothing}
-				${!this.noseek ? html`<vwc-scrub-bar @userScrubRequest="${({ detail }: { detail: number }) => this.currentTime = detail * this._duration}" class="scrubber"></vwc-scrub-bar>` : nothing}
+				${this.timestamp ? html`<div class="playhead-position">${getTimeStampTemplate(this._playheadPosition, this._duration)}</div>` : nothing}
```

Comment:
it looks like `getTimeStampTemplate` is already wrapping its returned markup with `<div class="playhead-position">`

Reply:
Nice catch! Removed.
![image](https://user-images.githubusercontent.com/6459899/228809557-4f7a3af8-3f53-438a-9ad5-33d24a932aec.png)